### PR TITLE
Remove Java-like curly brackets

### PR DESCRIPTION
**Pull request by yahiheb**

### CODE REVIEW

File: NBitcoin.Altcoins/Argoneum.Internals/Gost.cs [hunk 1]
```diff
@@ -626,7 +626,8 @@ private void AddModulo512(byte[] a, byte[] b, byte[] c)
 
         private void AddXor512(byte[] a, byte[] b, byte[] c)
         {
-            for (int i = 0; i < 64; i += 8) {
+            for (int i = 0; i < 64; i += 8)
+			{
```
Comment (knocte) [inline NBitcoin.Altcoins/Argoneum.Internals/Gost.cs:630]: you're mixing tabs and spaces

File: NBitcoin.Altcoins/Argoneum.Internals/Phi2.cs [hunk 2]
```diff
@@ -37,9 +37,12 @@ public byte[] ComputeHash(byte [] input)
 
             hash = jh512.ComputeBytes(hash).GetBytes();
 
-            if ((hash[0] & 1) == 1) {
+            if ((hash[0] & 1) == 1)
+			{
```
Comment (knocte) [inline NBitcoin.Altcoins/Argoneum.Internals/Phi2.cs:41]: same here and in more places

File: NBitcoin.Altcoins/Argoneum.Internals/Gost.cs (hunk 1 above)
Comment (yahiheb) [reply NBitcoin.Altcoins/Argoneum.Internals/Gost.cs]: Fixed.

File: NBitcoin.Altcoins/Argoneum.Internals/Phi2.cs (hunk 2 above)
Comment (yahiheb) [reply NBitcoin.Altcoins/Argoneum.Internals/Phi2.cs]: Fixed.

**Pull request merged**